python-sdk/src/astro/files/base.py

```diff
@@ -22,27 +24,27 @@ class File:  # skipcq: PYL-W1641
     :param normalize_config: parameters in dict format of pandas json_normalize() function.
     """
 
-    template_fields = ("location",)
```

Comment:
@kaxil is this change to `template_fields` a breaking change in any sense?

I noticed it is now:
```
   template_fields = (
        "path",
        "conn_id",
    )
```
That makes sense, but I was wondering if there was a use case for a `location` to be a template field in the first place.

Should we update the changelog as part of this change to inform the users?

Reply:
Since `location` object just has `path` and `conn_id`, this will have no impact i.e should not change any behavior as now instead of `location`, we template `path` and `conn_id`.